src/test/operators/maintenance/create_table_test.cpp

```diff
@@ -0,0 +1,48 @@
+#include <memory>
+
+#include "base_test.hpp"
+#include "gtest/gtest.h"
+
+#include "operators/maintenance/create_table.hpp"
+#include "storage/storage_manager.hpp"
+#include "storage/table.hpp"
+
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+class CreateTableTest : public BaseTest {
+ public:
+  void SetUp() override {
+    column_definitions.emplace_back("a", DataType::Int, false);
+    column_definitions.emplace_back("b", DataType::Float, true);
+
+    create_table = std::make_shared<CreateTable>("t", column_definitions);
+  }
+
+  TableColumnDefinitions column_definitions;
+  std::shared_ptr<CreateTable> create_table;
+};
+
+TEST_F(CreateTableTest, NameAndDescription) {
+  EXPECT_EQ(create_table->name(), "Create Table");
+  EXPECT_EQ(create_table->description(DescriptionMode::SingleLine),
+            "Create Table 't' ('a' int NON-NULL, 'b' float NULLABLE)");
+  EXPECT_EQ(create_table->description(DescriptionMode::MultiLine),
+            "Create Table 't' ('a' int NON-NULL\n'b' float NULLABLE)");
+}
+
+TEST_F(CreateTableTest, Execute) {
```

Comment:
Could you please test the edge cases (Creating table that already exists, dropping table that doesn't)?